README.md

```
@@ -17,66 +17,63 @@ Diff Viewer renders `.diff` and `.patch` files inside VS Code with [diff2html](h
 - Reopen a diff with all files collapsed from the Explorer context menu.
 - Open referenced files directly from file headers and line numbers when those paths are available in the current workspace or by absolute path.
 
-## Demo
+## Demo 🎬
 
-### Without the extension
+### Without the extension 📄
 
 <p align="center">
   <img src="documentation/original.png" width="700">
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)